src/main/java/internet/shop/dao/daointerface/OrderDao.java

```diff
@@ -0,0 +1,4 @@
+package internet.shop.dao.daointerface;
```

Comment:
`daointerface` looks a bit clumsy to me
this is not one word

let's name it `interface`
or even better - have interfaces directly in `dao` folder
and their implementations - in `dao.impl` subfolder